### DISCUSSION TITLE
Can't send images?

### DISCUSSION THREAD

**Discussion by Anas099X** [devlikeapro/waha#317] [General]
I tried executing the following script but it doesnt send images
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/99427334/5d0ca06c-b41b-4f01-a91b-5d97d7a11514)

**Comment by allburov** [comment]
and what is the error you get back?

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by allburov** [reply]
what version do you use? You can check it by `GET /api/version`

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by Anas099X** [reply]
version: 2024.5.2

**Reply by allburov** [reply]
The full content would help better here :)
And the response for the example you send

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by Anas099X** [reply]
{"version":"2024.5.2","engine":"WEBJS","tier":"CORE","browser":"/usr/bin/chromium"}
also, I had no errors show up in the console

**Reply by Anas099X** [reply]
ok so, this issue is also in sending files generally

**Reply by Anas099X** [reply]
Oh wait. I think I found the problem. sorry for interruption.(I didnt read the logs properly )

**Reply by allburov** [reply]
>"tier":"CORE",

This is the issue, correct

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
